app/views/repair_requests/new.html.erb

```diff
@@ -10,6 +10,25 @@
           New repair for <%= @property.address %>
         </h1>
 
+        <div class="hackney-warning-labels">
+          <% if @cautionary_contacts.present? %>
+          <table class="hackney-cautionary-contact-table" id="hackney-cautionary-contact-raise-a-repair">
```

Comment:
I wasn't sure of the best way to add only this margin to the table on the raise a repair page, so I went with id.

Reply:
there is a govuk class for this:
https://design-system.service.gov.uk/styles/spacing/

Reply:
also, it looks like this whole section is repeated on `properties#show`, isn't it? You should make a partial if that's the case.

Reply:
> https://design-system.service.gov.uk/styles/spacing/

Omg this is magic!

Reply:
> also, it looks like this whole section is repeated on `properties#show`, isn't it? You should make a partial if that's the case.

How would the varied margin work with a partial?